data/defaults.json

```diff
@@ -1,7 +1,7 @@
 {
   "architecture": "amd64",
-  "debian_mirror": "http://deb.debian.org/debian",
-  "debian_security_mirror": "http://deb.debian.org/debian-security",
+  "debian_mirror": "https://mirrors.ustc.edu.cn/debian/",
```

Comment:
What is the motivation for this change? Using a specific third-party mirror, not affiliated with the Debian project, needs really strong motivation.